deployment-aws/puppet/mevapi/manifests/cloudwatch_agent.pp

```diff
@@ -0,0 +1,48 @@
+class mevapi::cloudwatch_agent () {
+
+  if $mevapi::platform == 'aws' {
+
+    $cloudwatch_agent_dir = '/opt/aws/amazon-cloudwatch-agent'
+
+    file { '/tmp/amazon-cloudwatch-agent.deb':
+        ensure => present,
+        source => 'https://s3.amazonaws.com/amazoncloudwatch-agent/ubuntu/amd64/latest/amazon-cloudwatch-agent.deb',
+        mode   => '0550',
+        owner  => $mevapi::app_user
+    }
+
+    file { '/tmp/amazon-cloudwatch-agent.deb.sig':
+        ensure => present,
+        source => 'https://s3.amazonaws.com/amazoncloudwatch-agent/ubuntu/amd64/latest/amazon-cloudwatch-agent.deb.sig',
```

Comment:
I'd factor out the URL since it's used more than once, for example:
```puppet
$cloudwatch_agent_url = 'https://s3.amazonaws.com/amazoncloudwatch-agent/ubuntu/amd64/latest/amazon-cloudwatch-agent.deb'
```